rust/src/container.rs

```diff
@@ -78,6 +78,10 @@ struct ContainerEncapsulateOpts {
     #[clap(long)]
     /// Output content metadata as JSON
     write_contentmeta_json: Option<Utf8PathBuf>,
+
+    /// Compare OCI layers of current build with another(imgref)
+    #[clap(name = "compare-with-build", long, short)]
+    compare_with_build: Option<String>,
```

Comment:
I wonder if this is more convenient as a separate CLI command that can take two arbitrary builds?  i.e. `rpm-ostree container compare-images`?  Or hmm, I guess if we're keeping it independent of RPM content it could live in ostree even as `ostree container compare-images` or so?